{"code":8000,"codeName":"AtlasError","connectionGeneration":0,"errorLabelSet":{},"errorResponse":{"code":8000,"codeName":"AtlasError","errmsg":"bad auth : authentication failed","ok":0},"level":"error","message":"Failed to connect to mongodb bad auth : authentication failed","ok":0,"service":"identity-service","stack":"MongoServerError: bad auth : authentication failed\n    at Connection.sendCommand (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/mongodb/src/cmap/connection.ts:551:17)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Connection.command (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/mongodb/src/cmap/connection.ts:625:22)\n    at async continueScramConversation (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/mongodb/src/cmap/auth/scram.ts:187:13)\n    at async executeScram (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/mongodb/src/cmap/auth/scram.ts:114:3)\n    at async ScramSHA1.auth (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/mongodb/src/cmap/auth/scram.ts:60:12)\n    at async performInitialHandshake (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/mongodb/src/cmap/connect.ts:164:7)\n    at async connect (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/mongodb/src/cmap/connect.ts:43:5)","timestamp":"2025-05-17T14:04:23.674Z"}
{"code":"ENOTFOUND","hostname":"_mongodb._tcp.cluster0.txibm.mongodb.net","level":"error","message":"Failed to connect to mongodb querySrv ENOTFOUND _mongodb._tcp.cluster0.txibm.mongodb.net","service":"identity-service","stack":"Error: querySrv ENOTFOUND _mongodb._tcp.cluster0.txibm.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:294:17)","syscall":"querySrv","timestamp":"2025-05-17T15:02:04.498Z"}
{"0":"r","1":"e","2":"a","3":"s","4":"o","5":"n","6":":","7":" ","level":"error","message":"unhandled Rejection at:","service":"identity-service","timestamp":"2025-05-17T15:07:39.399Z"}
{"0":"r","1":"e","2":"a","3":"s","4":"o","5":"n","6":":","7":" ","level":"error","message":"unhandled Rejection at:","service":"identity-service","timestamp":"2025-05-17T15:07:39.405Z"}
{"0":"r","1":"e","2":"a","3":"s","4":"o","5":"n","6":":","7":" ","level":"error","message":"unhandled Rejection at:","service":"identity-service","timestamp":"2025-05-17T15:10:36.316Z"}
{"0":"r","1":"e","2":"a","3":"s","4":"o","5":"n","6":":","level":"error","message":"unhandled Rejection at:","service":"identity-service","timestamp":"2025-05-17T16:43:48.519Z"}
{"level":"error","message":"Internal Server Error secretOrPrivateKey must have a value","service":"identity-service","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/node_modules/jsonwebtoken/sign.js:111:20)\n    at generateTokens (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/src/utils/generateToken.ts:6:27)\n    at registerUser (/data/data/com.termux/files/home/Advanced-Express/social-media-microservices/identity-service/src/controllers/identity.controller.ts:29:63)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","timestamp":"2025-05-17T17:08:07.714Z"}
